helpers/validation.js

```diff
@@ -9,3 +10,30 @@ export const isObjectId = (value, helpers) => {
   // Return the value unchanged
   return value;
 };
+
+export const idValidationSchema = Joi.string().custom(isObjectId, 'Invalid id');
+
+export const addressValidationSchema = Joi.object().keys({
+  country: Joi.string().required(),
+  city: Joi.string().required(),
+  street: Joi.string().required(),
+  coordinate: Joi.object().keys({
+    lat: Joi.number().min(-90).max(90).required(),
+    lng: Joi.number().min(-180).max(180).required(),
+  }),
+});
+
+// Phone number should follow the pattern, for example: +38(050)1234567
```

Comment:
don't forget to provide this pattern on the UI, so user can understand what we are waiting him to pass